### ISSUE TITLE
Event Parameters Blocked on Stock SDK Install

### ISSUE THREAD

**Issue by danhalliday** [facebook/facebook-ios-sdk#2465]
### Checklist before submitting a bug report

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](https://github.com/facebook/facebook-ios-sdk/blob/main/CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

### Xcode version

15.2

### Facebook iOS SDK version

17.0.2

### Dependency Manager

SPM

### SDK Framework

Core

### Goals

Create a campaign which optimises for the `Subscribe` standard event.

### Expected results

I integrate the Facebook iOS SDK as per the official docs, I release my new app version, I see app events are coming in, and I'm able to create a new campaign that optimises for the `Subscribe` event.

### Actual results

In events manager, I see an 'event parameters blocked' diagnostic message. The summary states I'm sending personal data with an event parameter `fb_content_title` in several payment events such as `fb_mobile_purchase`.

When examining the `FBSDK` log output, I see that the offending parameter is only sending the names of my in-app purchase products, not personal data.

Nobody in support in the Facebook Business system can tell me anything about this. Nobody there seems to know the first thing about the iOS SDK. This is a 100% vanilla install, as per the docs. I'm not sending any custom events, just the stuff the SDK tracks automatically. What am I supposed to do?

### Steps to reproduce

Just do a standard SDK integration in an iOS app and release it to the App Store. Go to events manager and see the incorrectly blocked events.

### Code samples & details

```swift
This isn't a code issue.
```

**Comment by Zhiyi-Zhang** [comment]
Hi! After reviewing this issue, we're closing it because it appears to be a **server-side / platform issue** rather than a bug in the iOS SDK source code.

**Specific reason:** Server-side issue. Events Manager blocks fb_content_title parameter as 'personal data' even though it's a product name.

The Facebook iOS SDK is a client-side library that sends requests to Facebook's servers. Issues related to server responses, the Events Manager dashboard, Graph API behavior, app configuration, or Facebook platform policies are outside the scope of this SDK repository.

For platform-related issues, please use:
- [Meta for Developers Support](https://developers.facebook.com/support/)
- [Meta Business Help Center](https://www.facebook.com/business/help)

If you believe there's an SDK-side code bug contributing to this, please open a new issue with specific SDK code references and reproduction steps. Thanks!
